packages/cli/src/commands/upgrade.js

```diff
@@ -144,7 +144,7 @@ export const handler = async ({ dryRun, tag, verbose, dedupe }) => {
 }
 async function yarnInstall({ verbose }) {
   try {
-    await execa('yarn install', ['--force', '--non-interactive'], {
+    await execa('yarn install', ['--check-cache'], {
```

Comment:
The `--force` and `--non-interactive` options don't exist anymore, but `--check-cache` does what we want (for force at least). And `--non-interactive` doesn't seem to be a concern anymore.

Reply:
Why do you need to check the cache?

Reply:
@merceyz I was just porting the old command, trying to emulate `--force` (the docs say it "refetches all packages, even ones that were previously installed", which sounds similar to `--check-cache`: "Always refetch the packages and ensure that their checksums are consistent").

But I'm not sure that we need to do that anymore. @thedavidprice was there a reason we were using `--force`?

Reply:
`--check-cache` is to more to verify that the cache hasn't been tampered with in [zero-installs](https://yarnpkg.com/features/zero-installs) situations